Scripts/RestorePackages.bat

```diff
@@ -11,7 +11,8 @@ IF NOT EXIST %nuget% (
 
 %nuget% restore %VFS_SRCDIR%\GVFS.sln || exit /b 1
 
-dotnet restore %VFS_SRCDIR%\GVFS\GVFS.Common\GVFS.Common.csproj || exit /b 1
-dotnet restore %VFS_SRCDIR%\GVFS\GVFS.Virtualization\GVFS.Virtualization.csproj || exit /b 1
+SET VCTargetsPath=C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140

```

Comment:
This is needed for the .vcxproj's in our solution. They import .props files in this directory which is only set by the Framework version of msbuild. These projects won't actually restore anything under .NET Core, but the whole process will fail if they can't resolve their Imports.

Reply:
I'll rework this (and other similar patterns) to pass this variable directly into msbuild/dotnet rather than setting environment variables per an offline discussion about this pattern.